odd_collector/adapters/airbyte/api.py

```diff
@@ -0,0 +1,69 @@
+import json
+import requests
+import logging
+from typing import List
+
+
+class ApiGetter:
+    """Class intended for retrieving data from Airbyte API"""
+
+    __WORKSPACE_IDS = []
+    __CONNECTIONS = []
+
+    def __init__(self, host: str, port: int) -> None:
+        self.__base_url = f"http://{host}:{port}/api/v1"
+        self.__workspaces_request = f"{self.__base_url}/workspaces/list"
+        self.__connections_request = f"{self.__base_url}/connections/list"
+        self.__source_definition_request = f"{self.__base_url}/sources/get"
+        self.__destination_definition_request = f"{self.__base_url}/destinations/get"
+        self.__headers = {
+            "Content-type": "application/json",
+            "Accept": "application/json",
+        }
+
+    def get_all_workspaces(self) -> List[str]:
+        try:
+            response = requests.post(self.__workspaces_request).json()
+            workspaces = response["workspaces"]
+            for workspace in workspaces:
+                self.__WORKSPACE_IDS.append(workspace["workspaceId"])
```

Comment:
Same question about `self.__WORKSPACE_IDS`.
 I think you can just return new array .
```python
 return [workspace["workspaceId"] for workspace in workspaces]
```